src/lib/sqlite-database-integration-release.ts

```diff
@@ -0,0 +1,3 @@
+export const SQLITE_DATABASE_INTEGRATION_VERSION = 'v2.1.17-alpha';
+
+export const SQLITE_DATABASE_INTEGRATION_RELEASE_URL = `https://github.com/Automattic/sqlite-database-integration/archive/refs/tags/${ SQLITE_DATABASE_INTEGRATION_VERSION }.zip`;
```

Comment:
We could consider moving these constants to `src/constants.ts`
Not a strong opinion.

Reply:
Done [7948cc9](https://github.com/Automattic/studio/pull/950/commits/7948cc93589fdc328b3418d8fdccea78a37ca53e)